cmd/gossamer/main_test.go

```diff
@@ -268,7 +268,7 @@ func TestBuildSpecCommandWithOutput(t *testing.T) {
 	buildSpecCommand := runTestGossamer(t,
 		"build-spec",
 		"--raw",
-		"--genesis-spec", utils.GetGssmrGenesisPathTest(t),
+		"--genesis-spec", utils.GetGssmrGenesisRawPathTest(t),
```

Comment:
nit why the change? 🤔

Reply:
Pretty sure it was wrong before.  The raw spec contains the code.